assets/ucsd.yaml

```diff
@@ -0,0 +1,50 @@
+---
+
+- type: model
+  name: Baize
+  # General
+  organization: UC San Diego, Sun Yat-sen University
+  description: >
+    Baize is an open-source chat model that uses parameter-efficient tuning (specifically,
+    LoRA) to enhance the LLaMA model. It uses 100k dialogs generated by letting
+    ChatGPT chat with itself. It also uses Alpaca's data to improve its performance.
+  created_date:
+    value: 2023-04-03
+    explanation: >
+      The date the [[paper]](https://arxiv.org/abs/2304.01196) was released.
+  url: https://arxiv.org/pdf/2304.01196.pdf
+  model_card: ''
+  modality: text
+  size:
+    value: 30B parameters (dense model)
+    explanation: Available in 7B, 13B and 30B sizes.
+  analysis: ''
+  # Construction
+  dependencies:
```

Comment:
should there also be a dependency on gpt-3.5-turbo (based on the description and my reading of page 2/Section 3)?